Makefile.docker

```diff
@@ -28,7 +28,7 @@ lint:
 	${CDWD} && go fmt ${NO_VENDOR}
 	golint ${SRC}
 
-test:
+test: vendor
```

Comment:
Actually, I'm thinking that we should make sure that we're asserting the dependency here in only one makefile to avoid confusion. I'll move this item into https://github.com/joyent/containerbuddy/pull/95.